x-pack/plugin/security/qa/multi-cluster/src/javaRestTest/java/org/elasticsearch/xpack/remotecluster/AbstractRemoteClusterSecurityTestCase.java

```diff
@@ -86,14 +86,27 @@ protected Settings restClientSettings() {
     }
 
     protected void configureRemoteClustersWithApiKey(String indicesPrivilegesJson) throws IOException {
+        final String encodedRemoteAccessApiKey = createRemoteAccessApiKey(indicesPrivilegesJson);
+
+        // Update remote cluster settings on QC with the API key
+        updateClusterSettings(
+            Settings.builder()
+                .put("cluster.remote.my_remote_cluster.mode", "proxy")
+                .put("cluster.remote.my_remote_cluster.proxy_address", fulfillingCluster.getRemoteClusterServerEndpoint(0))
+                .put("cluster.remote.my_remote_cluster.authorization", encodedRemoteAccessApiKey)
+                .build()
+        );
+    }
+
+    protected String createRemoteAccessApiKey(String indicesPrivilegesJson) throws IOException {
         // Create API key on FC
         final var createApiKeyRequest = new Request("POST", "/_security/api_key");
         createApiKeyRequest.setJsonEntity(Strings.format("""
             {
               "name": "remote_access_key",
               "role_descriptors": {
                 "role": {
-                  "cluster": ["cluster:monitor/state"],
+                  "cluster": ["cluster:admin/remote_cluster/*"],
```

Comment:
This can be packaged with a named privilege, e.g. `remote_cluster_client`, to hide the implementation details.